styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -417,13 +443,17 @@ public void updateNextNaturalTrigger(WorkflowId workflowId, TriggerInstantSpec t
         .map(entity -> entity.getKey().getName())
         .map(name -> activeWorkflowInstanceKey(datastore.newKeyFactory(), name))
         .collect(toList());
+    stats.recordDatastoreEntityReads(KIND_ACTIVE_WORKFLOW_INSTANCE_INDEX_SHARD_ENTRY, keys.size());
 
     // Strongly consistently read values for the above keys
-    return Lists.partition(keys, MAX_NUMBER_OF_ENTITIES_IN_ONE_BATCH_READ).stream()
+    final Map<WorkflowInstance, RunState> states = Lists.partition(keys, MAX_NUMBER_OF_ENTITIES_IN_ONE_BATCH_READ)
```

Comment:
why the change here?

Reply:
This change is no longer needed, will revert.